addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -39,6 +39,15 @@
                     <t t-call="website.form_field_description"/>
                 </div>
             </div>
+            <t t-elif="field.formatInfo.labelPosition == 'after'">
+                <div class="d-flex">
```

Comment:
Rather than `flex`, I think that we should use a `.checkbox > .form-check` structure here 🤔
You can see it implemented already for the "_Multiple Checkboxes_" field type.
ref: https://getbootstrap.com/docs/5.3/forms/checks-radios/#checks